### ISSUE TITLE
Translated language name in the dropdown

### ISSUE THREAD

**Issue by brmassa** [chidiwilliams/buzz#907]
use the translated language name in the dropdown so if you accidentally click in Mandarin, you can still read "English" or "Español" and return back! 

instead:

- Arabic
- Bengali
- English
- Hindi
- Marathi
- Mandarin Chinese
- Punjabi
- Portuguese
- Russian
- Spanish

it should be presented as:

- English
- Español
- Português 
- Русский
- العربية
- मराठी
- हिन्दी
- বাংলা
- ਪੰਜਾਬੀ
- 普通话

**Comment by Chenghao999** [comment]
I agree that I also think we should add "language" to the performance because sometimes the translation is not one hundred percent true.

**Comment by raivisdejus** [comment]
Next version will have an option to switch the language of the UI in the preferences. This feature is already available in the latest development build https://chidiwilliams.github.io/buzz/docs/faq#9-where-can-i-get-latest-development-version

Language names are also translatable now, but those have to be useful for the users of the selected UI language. If I am using the Buzz in Latvian these entries in the language menu mean nothing to me `ਪੰਜਾਬੀ`, `普通话`. I need them in my language to be able to understand what I want to switch.

I have also improved language detection, so hopefully everybody gets Buzz in their language. Accidental misclicks that end up in UI in some foreign language should be very rare and hopefully users can re-install Buzz or trace their way around the UI and select the necessary language. Buzz has limited amount of menus, so I expect this to not be a big problem.

**Comment by Chenghao999** [comment]
Could you change `普通话` to `简体中文`, which is batter? 
thx!
